docker/puppetserver-standalone/healthcheck.sh

```diff
@@ -3,12 +3,16 @@
 set -x
 set -e
 
-hostname=$(puppet config print certname) && \
+certname=$(ls /etc/puppetlabs/puppet/ssl/certs | grep --invert-match ca.pem) && \
+hostname=${certname%.*} && \
```

Comment:
This construct is not one I was familiar with either and had to look it up as well... 

> ${var%Pattern} Remove from $var the shortest part of $Pattern that matches the back end of $var.

But since we're matching everything with `.*`, I don't think I understand why the `%.*` is needed?

Reply:
@Iristyle we're stripping off the file extension here, that `.` isn't a meta-character it's the literal `.`, so `puppet.test.pem` would go to `puppet.test`

Reply:
ah, cool. I hadn't. That seems more straightforward, will switch to that